deploy.sh

```diff
@@ -590,10 +591,17 @@ function apply {
 # Destroy the cluster
 #-------------------------------------------------------------------------
 function destroy {
-  precheck
-  log "Running terraform destroy... please wait"
-  retry_terraform 2 destroy "$vars -input=false"
-  success "Done! destroy commmand completed"
+  if [ -d $ARTIFACTS_DIR ] && [ -f ./"$ARTIFACTS_DIR"/terraform.tfstate ]; then
+    if [[ $($TF state list -state=./"$ARTIFACTS_DIR"/terraform.tfstate | wc -l) -gt 0 ]]; then
+      [[ $FORCE_DESTROY == 0 ]] && question "Are you sure you want to proceed with destroy?" "yes"
+      if [[ "${value}" == "yes" || $FORCE_DESTROY == 1 ]]; then
```

Comment:
What is the value of `value` incase not force destroy? could be the answer of the last question.